llvm/lib/DebugInfo/DWARF/DWARFVerifier.cpp

```diff
@@ -1598,7 +1588,7 @@ unsigned DWARFVerifier::verifyNameIndexEntries(
   for (; EntryOr; ++NumEntries, EntryID = NextEntryID,
                                 EntryOr = NI.getEntry(&NextEntryID)) {
 
-    std::optional<uint64_t> CUIndex = EntryOr->getCUIndex();
+    std::optional<uint64_t> CUIndex = EntryOr->getRelatedCUIndex();
     std::optional<uint64_t> TUIndex = EntryOr->getLocalTUIndex();
```

Comment:
Should the API be renamed from `getLocalTUIndex()` be `getTUIndex()` since the returned index can be either local or foreign type unit index now?